config/environments/test.rb

```diff
@@ -1,4 +1,16 @@
 Rails.application.configure do
+  {
+    ENGLISH_URI:  '/rest/v1/language/100/',
```

Comment:
I think some of the env vars could be added to `config/environments/production.rb`in a similar fashion (at least the language URIs) - the panel on AWS is pretty crowded. Maybe in another pull request.